README.md

```diff
@@ -32,6 +32,7 @@ The current feature list includes:
 * [x] Firewall (partial)
 * [ ] [Keyless SSL](https://blog.cloudflare.com/keyless-ssl-the-nitty-gritty-technical-details/)
 * [x] [Load Balancing](https://blog.cloudflare.com/introducing-load-balancing-intelligent-failover-with-cloudflare/)
+* [x] [Logpush Jobs](https://developers.cloudflare.com/logs/logpush/) (partial)
```

Comment:
Any reason why only partial support? What is blocking full support?

Reply:
Missing:
https://api.cloudflare.com/#logpush-jobs-get-ownership-challenge
https://api.cloudflare.com/#logpush-jobs-validate-ownership-challenge
https://api.cloudflare.com/#logpush-jobs-check-destination-exists

It's not needed for the root cause of PR :)

Reply:
It looks easy, I will add it soon

Reply:
Can we stretch here and add them? I think all three are relatively straight forward and would be great to have full support from the beginning.